prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -263,8 +240,43 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
     }
   }
 
+  public GetEmailsGroup(careSettingCode: number) {
```

Comment:
Please add return type information; that's the project's convention despite compiler acceptance

Reply:
fixed